Base/QTGUI/qSlicerWebPythonProxy.cxx

```diff
@@ -37,6 +37,7 @@ qSlicerWebPythonProxy::qSlicerWebPythonProxy(QObject *parent)
   : QObject(parent)
 {
   this->pythonEvaluationAllowed = false;
+  this->debugEvalRequestsState = false;
```

Comment:
This name should be improved.
1. A "...State" cannot be Boolean (true/false). Either its type can be changed to an enum, or its type can be kept as a Boolean but then renamed. 
2. "debugEvalRequests" is a bit too specific. If there is something wrong in the class and we want to debug it then it is simpler and more convenient to have only one "debug" flag that enables more detailed logging overall.

CTK  already uses the simple `verbose` property in a number of classes to enable/disable detailed logging, so it could be used here. Or we could introduce a more self-explaining `detailedLoggingEnabled` flag (similarly to the DICOM plugins).